js/plugins/conviva.js

```diff
@@ -246,7 +247,11 @@ var ConvivaAnalyticsPlugin = function(framework)
       // A human-readable identifier for your application.
       // Very helpful to filter traffic and compare performance for different builds of
       // the video application.
-      // contentMetadata.applicationName = "JS SDK Sample Application 1.2.3";
+      var appName = convivaMetadata["applicationName"];
+      if (_.isString(appName))
+      {
+        contentMetadata.applicationName = appName;
```

Comment:
It's not a required field by the Conviva SDK, but it is a required field for the SkyNews metadata we will be testing against.